testsuite/lattice_boltzmann/test_LatticeBoltzmann.py

```diff
@@ -27,18 +27,19 @@
 
 import unittest
 
-runSteps = 500
+runSteps = 600
 temperature = 1.0
-Ni = 5
+Nx = 32
+Ny = Nz = 4
 initDen = 1.
 initVel = 0.
 initVelSin = 0.1
 
 class TestPureLB(unittest.TestCase):
     def setUp(self):
         # set up system
-        global Ni, temperature
-        system, integrator = espressopp.standard_system.LennardJones(0, box=(Ni, Ni, Ni), temperature=temperature)
+        global temperature
```

Comment:
why do you need those global variables?  you could set them in the `setUp` method and then use `self.temperature`, `self.runSteps`

Reply:
thanks, I've corrected this. But this code was partially overwritten in the last commit